capstone/src/main/java/com/google/sps/servlets/RegisterDriver.java

```diff
@@ -9,14 +9,17 @@
 import java.util.Arrays;
 import com.google.sps.data.Driver;
 import com.google.sps.data.Rider;
-import com.google.sps.data.Car;
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
 import com.google.gson.Gson;
+import org.json.JSONArray;
```

Comment:
Sort imports.

Reply:
I was not sure where to put the org.json, not sure how to sort them, alphabetically maybe?

Reply:
yes, alphabetically. We used to do some grouping, like putting "org" ones last with a blank line before, but nowadays just sort all imports alphabetically.